go/libraries/doltcore/sqle/procedures_table.go

```diff
@@ -94,6 +101,88 @@ func DoltProceduresGetOrCreateTable(ctx *sql.Context, db Database) (*WritableDol
 	return tbl.(*WritableDoltTable), nil
 }
 
+// migrateDoltProceduresSchema migrates the dolt_procedures system table from a previous schema version to the current
+// schema version by adding any columns that do not exist.
+func migrateDoltProceduresSchema(ctx *sql.Context, db Database, oldTable *WritableDoltTable) (newTable *WritableDoltTable, rerr error) {
```

Comment:
I believe we do need explicit migration, otherwise the `dolt_procedures` table won't ever get the new column. `dolt_schemas` does also [have explicit migration code](https://github.com/dolthub/dolt/blob/75b80fd476a7b3d94fd33a627e6f7ae5ecd61ddd/go/libraries/doltcore/sqle/schema_table.go#L108).